PYME/Acquire/Hardware/Camera.py

```diff
@@ -33,43 +33,73 @@
 import logging
 logger = logging.getLogger(__name__)
 
-def check_mapexists(mdh, type='dark'):

```

Comment:
I like having this as a stand-alone function, rather than part of a camera mixin, as it would let us use it, e.g. from the LMAnalysis plugin for dh5view if map locations haven't been recorded at acquisition time, or if their locations have changed as we're analysing on a different computer.

Ultimately it probably belongs somewhere else (a refactored and renamed gen_scmod_maps.py?), but let's not worry about that for now.